user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md

```diff
@@ -7,6 +7,10 @@ uid: Security_Cassandra_TLS
 > [!TIP]
 > If you do not want the hassle of maintaining the DataMiner storage databases yourself, we recommend using [DataMiner Storage as a Service](xref:STaaS) instead.
 
+> [!NOTE]
+> We strongly advise to use certificates based on hostname and not IP. These hostnames should then be used in the configuration of db.xml that you can find in Connecting with DataMiner below. This recommendation is for self signed  certificates and purchased certifcates.
+> In case you would want to use the IP in the certificate and db.xml, you have to make sure that you can resolve the IP to a hostname on the network, or that the SAN is correctly set in the certificate.
```

Comment:
Even with SAN, there should be some DNS resolution linking the IP to the CN of the cert (cassandra always does a hostname lookup)